WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/LegacyCollectOrderPaymentUseCaseTests.swift

```diff
@@ -110,27 +110,6 @@ final class LegacyCollectOrderPaymentUseCaseTests: XCTestCase {
     }
 
     // MARK: Success alert actions
-
-    func test_printing_receipt_from_collectPayment_success_alert_tracks_receiptPrintTapped_event() throws {
```

Comment:
I deleted this test case because we have it already implemented in `ReceiptActionCoordinatorTests`, where it feels more natural.